frontend/packages/noobaa-storage-plugin/integration-tests/tests/objectServiceDashboard.scenario.ts

```diff
@@ -0,0 +1,214 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { untilNoLoadersPresent } from '@console/internal-integration-tests/views/crud.view';
+import { waitFor } from '@console/ceph-storage-plugin/integration-tests/utils/helpers';
+import {
+  OCP_HEALTH_ICON_COLORS,
+  SECOND,
+} from '@console/ceph-storage-plugin/integration-tests/utils/consts';
+import { click } from '@console/shared/src/test-utils/utils';
+import {
+  efficiencyValue,
+  healthOfMCG,
+  noobaaCount,
+  obcCount,
+  resourceProviders,
+  savingsValue,
+  resiliencyOfMCG,
+  CapacityBreakdown,
+  PerformanceCard,
+  unifiedHealthButton,
+  POPOVER_GREEN_COLOR,
+  unifiedDataResiliencyButton,
+  mcgPopover,
+  rgwPopover,
+} from '../views/noobaaDashboardPage.view';
+import { testBucket } from '../mocks/obcData';
+import { RGW_PROVISIONER } from '../utils/consts';
+
+const loadAndWait = async () => {
+  await browser.get(`${appHost}/dashboards/object-service`);
+  await browser.wait(until.and(untilNoLoadersPresent));
+};
+
+const storageClasses = JSON.parse(execSync('oc get StorageClasses -o json').toString());
+const isRGWPresent = storageClasses.items.some((sc) => sc.provisioner === RGW_PROVISIONER);
+
+describe('Tests Buckets Card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Test at least one noobaa bucket is present', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
```

Comment:
This can be removed as this is already added inside `loadAndWait` function.

Reply:
As I told in the PR that initially wrote this. It for some reason breaks without this.